src/int.cr

```diff
@@ -172,7 +172,7 @@ struct Int
   def %(other : Int)
     if other == 0
       raise DivisionByZeroError.new
-    elsif (self ^ other) >= 0
+    elsif (self.sign ^ other.sign) >= 0
```

Comment:
Yes, but what about the equivalent expression `(a<0) == (b<0)`?
https://carc.in/#/r/79lf

Reply:
You are right, I was overcomplicating things. Changed, thanks!